src/core/server/ui_settings/ui_settings_service.ts

```diff
@@ -60,6 +61,8 @@ export class UiSettingsService
 
     savedObjects.registerType(uiSettingsType);
     registerRoutes(http.createRouter(''));
+    this.register(getCoreSettings());
```

Comment:
I kept the registration internally to the service. Tell me if it should be done from elsewhere.